deepspeed/utils/zero_to_fp32.py

```diff
@@ -483,6 +530,7 @@ def get_fp32_state_dict_from_zero_checkpoint(checkpoint_dir, tag=None, exclude_f
         - ``checkpoint_dir``: path to the desired checkpoint folder
         - ``tag``: checkpoint tag used as a unique identifier for checkpoint. If not provided will attempt to load tag in 'latest' file. e.g., ``global_step14``
         - ``exclude_frozen_parameters``: exclude frozen parameters
+        - ``lazy_merge``: a more memory-efficient feature
```

Comment:
Provide brief description of why more memory-efficient, and perhaps mention important usage concepts like pseudo tensors and `contiguous()`.
 
Also, please add a unit test.

Reply:
This would be a good place for a unit test: https://github.com/microsoft/DeepSpeed/blob/0e11b081be237f9b1867a3af88479a23de11345f/tests/unit/checkpoint/test_shared_weights.py#L48

Reply:
thanks